avocado/core/data_dir.py

```diff
@@ -212,7 +220,7 @@ def __del__(self):
 _tmp_tracker = _TmpDirTracker()
 
 
-def get_tmp_dir():
+def get_tmp_dir(basedir=None, keep_tmp=False):
```

Comment:
Because the user did not specify it. We can then check whether the user really specified to disable (`False`) or whether he left it as default (`None`).